src/Mod/Spreadsheet/App/PropertySheetPyImp.cpp

```diff
@@ -23,14 +23,31 @@
 
 #include "PreCompiled.h"
 
-#include "PropertySheet.h"
+#include "Mod/Spreadsheet/App/PropertySheet.h"
 
 // inclusion of the generated files (generated out of PropertySheetPy.xml)
 #include "PropertySheetPy.h"
 #include "PropertySheetPy.cpp"
 
 using namespace Spreadsheet;
 
+struct PropertySheetPyInit {
```

Comment:
We do not need this hack to support Python's mapping protocol. We can properly define it in PropertySheetPy.xml and then implement the function mapping_subscript. As an example VectorPy.xml can be used.